source/content.js

```diff
@@ -6,6 +6,10 @@ import autoLoadNewTweets from './features/auto-load-new-tweets';
 // Add global for easier debugging
 window.$ = $;
 
+function cleanPromotedTweets() {
+	$('#stream-items-id .promoted-tweet').remove();
```

Comment:
You don't need `#stream-items-id`. The class is specific enough.

Reply:
And you should probably remove the parent item, as that's the actual container, just to be safe. So:

```js
$('.promoted-tweet').parent().remove();
```